src/app/backing/components/AllocationBar/AllocationBarSegment.tsx

```diff
@@ -112,16 +115,44 @@ export const AllocationBarSegment = ({
       `.trim()}
     >
       {/* DRAG HANDLE of the size of the segment */}
-      {isDraggable && <AllocationBarDragHandle attributes={attributes} listeners={listeners} />}
-
-      {
-        <AllocationBarSegmentPercent
-          value={value}
-          totalValue={totalValue}
-          valueDisplay={valueDisplay}
-          showDots={showDots}
-        />
-      }
+      {isDraggable && (
+        <div className="group relative w-full h-full">
+          <AllocationBarDragHandle attributes={attributes} listeners={listeners} />
+
+          <div className="absolute bottom-full left-1/2 -translate-x-1/2 mb-2 z-50 pointer-events-none opacity-0 invisible group-hover:opacity-100 group-hover:visible transition-opacity duration-200">
+            <div className="rounded-sm bg-v3-text-80 text-v3-bg-accent-60 px-2 py-1 text-xs font-normal shadow-lg font-rootstock-sans">
+              <AllocationBarTooltip
+                isTemporary={item.isTemporary}
+                builderAddress={item.key}
+                currentBacking={value}
+                pendingBacking={initialValue}
```

Comment:
current and pending should on the other way around, with current set to initial and pending set to value